GitHubContents.ps1

```diff
@@ -340,6 +342,8 @@ filter Set-GitHubContent
 
         [string] $AuthorEmail,
 
+        [string] $ContentPath,
```

Comment:
`$ContentPath` should be mutually exclusive with `$Content`.  You can introduce an additional `ParameterSet` to help control that.  It's worthy of a separate example in the documentation above as well.

Reply:
Fixed